inlang/source-code/versioned-interfaces/language-tag/src/interface.ts

```diff
@@ -3,10 +3,14 @@ import { Type, type Static } from "@sinclair/typebox"
 /**
  * Follows the IETF BCP 47 language tag schema.
  *
- * // TODO: add (regex) validation
- *
  * @see https://www.ietf.org/rfc/bcp/bcp47.txt
  * @see https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry
  */
 export type LanguageTag = Static<typeof LanguageTag>
-export const LanguageTag = Type.String()
+/**
+ * Follows the IETF BCP 47 language tag schema with modifications.
+ * @see REAMDE.md file for more information.
+ */
+export const LanguageTag = Type.RegExp(
```

Comment:
The type is wrong.  A language tag is not a regex. It's a string that you want to validate with a regex

Type.String({ 
 pattern: regex
})

Reply:
Oh, really? Then I'm misunderstanding this doc.

<img width="798" alt="Bildschirmfoto 2023-09-22 um 16 39 42" src="https://github.com/inlang/monorepo/assets/34959078/23997dc2-8b63-4ef4-9730-73a2f35d427e">

Reply:
Oh you're right. I would use string regardless to add descriptions and match with JSON Schema.

Reply:
@samuelstroschein Can you clarify? Writing it as you told gives ts errors.
Added tests 👍 

<img width="680" alt="Bildschirmfoto 2023-09-22 um 17 40 53" src="https://github.com/inlang/monorepo/assets/34959078/4e2cf173-3833-4d7d-a34e-42950850495b">